### PR TITLE
Fix zero time bug (v4.0.0)

### PR DESCRIPTION
**Pull request by rajaeinet**
**Fix Zero-Time Bug** (like issue [#107](https://github.com/wordpress-parsi/wp-parsidate/issues/107)) and Duplicate date in activity panel (reported [here](https://wordpress.org/support/topic/%d9%85%d8%b4%da%a9%d9%84-%d8%af%d8%b1-%d8%aa%d8%a7%d8%b1%db%8c%d8%ae-%d8%a8%d8%b9%d8%af-%d8%a7%d8%b2-%d8%a2%d9%be%d8%af%db%8c%d8%aa/))

These changes have been tested in:
- Posts: posts list, single page (`the_time()` & `get_the_time()`), relative times (... ago time)
- WooCommerce: products list, orders list
- WooCommerce: Jalali datepicker (product, order & coupon)
- parsi-date: Jalali Permalink (wpp_settingsconv_permalinks)
- ACF (Jalali Date & Time)
- Gutenberg Jalali Calendar

In addition, date and time numbers in the Dashboard are displayed with Persian digits.

### CODE REVIEW

File: includes/parsidate.php [hunk 1]
```diff
@@ -256,7 +256,7 @@ public function persian_date( $format, $date = 'now', $lang = 'per' ) {
 			}
 		}
 
-		if ( in_array( strtolower( $format ), [ 'u', 'timestamp' ] ) && $lang == 'per' ) {
+		if ( !in_array( strtolower( $format ), [ 'u', 'timestamp' ] ) && $lang == 'per' ) {
```
Comment (man4toman) [inline includes/parsidate.php]: This line already fixed, but thank you!

File: includes/fixes-dates.php [hunk 2]
```diff
@@ -15,7 +15,7 @@
 if ( get_locale() == 'fa_IR' && wpp_is_active( 'persian_date' ) ) {
 	add_filter( 'the_time', 'wpp_fix_post_time', 10, 2 );
 	add_filter( 'the_date', 'wpp_fix_post_date', 10, 2 );
-	add_filter( 'get_the_time', 'wpp_fix_post_date', 10, 2 );
+	add_filter( 'get_the_time', 'wpp_fix_post_time', 10, 2 );
```
Comment (man4toman) [inline includes/fixes-dates.php:18]: We use 'wpp_fix_post_date' instead 'wpp_fix_post_time' to prevent conflict in admin area.

Comment (rajaeinet) [reply includes/fixes-dates.php:18]: Which conflict do you mean?!
This is the main reason for zero time bug in posts list (`00:00:00 AM`) & duplicate dates (like attach image), According to my reviews.
In my short experiments, there was no conflicts, But it's better to test it yourself. (In the previous version, `wpp_fix_post_date` was used!)

Comment (man4toman) [reply includes/fixes-dates.php:18]: Thanks for description, we test it again in new version.

**Comment by man4toman** [comment]
@hamidrezayazdani Would you please check this pr with our latest version of changes?

**Comment by hamidrezayazdani** [comment]
@man4toman Sure, I'll be review the PR this week.

**Review by hamidrezayazdani — APPROVED** [review]

**Pull request merged**